Step 7: Dynamic portfolio section

leehaney254 opened the pull request:
## In this section we: 👍 
- created a dynamic portfolio section
- created the cards of the project dynamically
- created a pop-up window dynamically

leehaney254 committed:
Feat: Create the cards dynamically

leehaney254 committed:
Style: Add style to the first card

leehaney254 committed:
Style: Add spacing to card in mobile version

leehaney254 committed:
Style: Add margin to cards

leehaney254 committed:
Feat: Add condition to check mediaQuery

leehaney254 committed:
Feat: Add popup dynamically

leehaney254 committed:
Style: Add style to pop-up card

solorics committed:
add popup function

leehaney254 committed:
Style: Add style to desktop version cards

leehaney254 committed:
Feat: Add pop-up close button

leehaney254 committed:
Feat: Remove modal when cancelled

leehaney254 committed:
Style: Add margin to popup

leehaney254 committed:
Fix linter errors

leehaney254 committed:
Feat: Add blur background on popup

leehaney254 committed:
Fix linters errors

tufoinnkuo10 commented on app.js:
```diff
@@ -63,4 +63,268 @@ function fillData() {
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
```
- [x] Great job. please centralize your mobile version image so it looks like the figma version
- [ ] kindly adjust the tech buttons as well. also, ensure the see source and see live buttons are of same size. the button icons well adjusted as well. great job

yours
![Screenshot (604)_LI](https://user-images.githubusercontent.com/65298048/202849239-e3d032ed-4eb2-4d58-835a-5ff8e47d207c.jpg)
figma
![Screenshot (600)](https://user-images.githubusercontent.com/65298048/202849270-54b283b2-a3e7-43af-bebb-8ef17a6aae4f.png)

tufoinnkuo10 commented on app.js:
```diff
@@ -63,4 +63,268 @@ function fillData() {
+    desktopImg: './assets/Img-Placeholder.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
```
- [x] please ensure that you follow the figma design as much as possible. resize and position your image in the desktop popup so it takes the figma design.

your design
![Screenshot (605)_LI](https://user-images.githubusercontent.com/65298048/202849394-c0152ada-baae-49fa-8ad6-5cf12bee076e.jpg)
figma
![Screenshot (599)](https://user-images.githubusercontent.com/65298048/202849407-43016891-c41a-4e07-8173-9f0613b807c7.png)

tufoinnkuo10 reviewed (changes requested):
Hi @leehaney254 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

leehaney254 committed:
Style: Add style to popup badge

leehaney254 committed:
Style: Add style to pop-up button

leehaney254 commented:
Hello!
- I have resized and positioned my desktop image to follow the figma design. 💪 
- I have centralized my image to match that of figma.
- I have adjusted the tech buttons
- Fixed the source and live button 👍

leehaney254 committed:
Fix linters errors

Qoosim commented on app.js:
```diff
@@ -63,4 +63,274 @@ function fillData() {
+window.addEventListener('load', fillData);
+
+// create the card information
+const otherProjects = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
```
- [ ] Good job in implementing the suggestions from the previous reviewer. However, there is still one issue you need to fix on this part of the project. Some content of the page cut off at the bottom of your modal on the mobile. Kindly look into it and fix it.

### Your Design
![mobile-modal](https://user-images.githubusercontent.com/26597432/202853197-b6003028-2245-4ce4-af30-95986737d1ba.png)

Qoosim commented on app.js:
```diff
@@ -63,4 +63,274 @@ function fillData() {
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: './assets/rest-card.png',
+    desktopImg: './assets/Img-Placeholder.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
```
- [ ] The same issue affects the desktop version. Part of the modal is cut off at the bottom of the popup. Kindly look into it and fix it.

### Your Design
![desktop-modal](https://user-images.githubusercontent.com/26597432/202853274-c754f021-ce15-4224-81d1-36f60d62ad1f.png)

Qoosim reviewed (changes requested):
#   *Status: Required Changes* :recycle: 

Hi @leehaney254, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://user-images.githubusercontent.com/25227022/169399289-510de072-42e7-419b-8c2c-da9344bdd430.gif)


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

leehaney254 commented:
I have fixed the breaking code on mobile version 💪

leehaney254 committed:
Style: Fix break in modal popup

anagudelogu reviewed (approved):
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @leehaney254, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

- [ ] **[OPTIONAL]:** I noticed that your project has a horizontal overflow, I would suggest you look into it and fix it. Having a good-looking UI design will make sure future recruiters get more attracted to your work! 🤓 🛠️ 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
